cli/cmd/generate_aws.go

```diff
@@ -231,9 +271,9 @@ See help output for more details on the parameter value(s) required for Terrafor
 			// Parse passed in subaccounts (if any)
 			if len(GenerateAwsCommandExtraState.AwsSubAccounts) > 0 {
 				// validate consolidated_cloudtrail is enabled - otherwise this flag cannot be used
-				if ok, _ := cmd.Flags().GetBool("consolidated_cloudtrail"); !ok {
-					return errors.New("aws subaccounts can only be supplied with consolidated cloudtrail enabled")
-				}
+				//if ok, _ := cmd.Flags().GetBool("consolidated_cloudtrail"); !ok {
```

Comment:
nit: is this commented out block left in intentionally?

Reply:
Good spot, was commented out to check it worked, it did and should be removed